src/app.js

```diff
@@ -40,14 +39,17 @@ const useTransformedOutput = (code, decimalImpl) => {
   const [transformed, setTransformed] = useState("");
   const [transformationError, setTransformationError] = useState(null);
 
-  const prefixedCode = `
-    ${prefixes[decimalImpl]}
-    ${code}
-  `;
-
   useEffect(() => {
     const transformOutput = async () => {
       try {
+        const response = await fetch(preambles[decimalImpl]);
+        const preamble = await response.text();
+        const encodedPreamble = btoa(preamble);
+        const prefixedCode = `
+          import "data:text/javascript;base64,${encodedPreamble}";\n
```

Comment:
An alternative would be to just paste the preamble in here, instead of importing it via a data URI. I picked this because I'm guessing Babel doesn't transform it. But maybe we actually want that?

If I had been better at React, I also would've preferred to load the preamble file only when switching Decimal implementations, not on every transform. But I didn't figure out how to do that in an afternoon. I can make that change later if this approach seems good.

Reply:
I don't think we need Babel to transform the preambles, but I am also not certain that they need to be in the output at all. 

I'm temped to just load them directly in `src/runner/index.html`. Since we are using different libs, we can have the methods themselves use `instance of` to select the correct method, right? Or if not, there may be another way to conditionally load the script without adding it to the output. 

The reason I would like to avoid the latter is the transformed code being focused on what the user has added seems helpful.

Reply:
Got it. Do note that Babel prepends a bunch of junk before the user's code if you use any language features that it decides it needs to transform. For example, [here](http://localhost:4444/#eyJjb250ZW50IjoiY29uc3QgU1VQRVJTQ1JJUFRTID0ge1xuICAwOiAnXFx1MjA3MCcsXG4gIDE6ICdcXHUwMGI5JyxcbiAgMjogJ1xcdTAwYjInLFxuICAzOiAnXFx1MDBiMycsXG4gIDQ6ICdcXHUyMDc0JyxcbiAgNTogJ1xcdTIwNzUnLFxuICA2OiAnXFx1MjA3NicsXG4gIDc6ICdcXHUyMDc3JyxcbiAgODogJ1xcdTIwNzgnLFxuICA5OiAnXFx1MjA3OScsXG4gICctJzogJ1xcdTIwN2InLFxufTtcblxuLy8gRm9ybWF0IGEgbnVtYmVyIHVzaW5nIHNjaWVudGlmaWMgbm90YXRpb25cbmZ1bmN0aW9uIHNjaW5vKG51bSwgbWF4aW11bUZyYWN0aW9uRGlnaXRzKSB7XG4gIG51bSA9IERlY2ltYWwobnVtKTtcbiAgY29uc3QgZXhwb25lbnQgPSBNYXRoLmZsb29yKE1hdGgubG9nMTAoTWF0aC5hYnMobnVtKSkpO1xuICAvLyBGSVhNRTogRGVjaW1hbChleHBvbmVudCkgLT4gZXhwb25lbnRcbiAgLy8gRklYTUU6IERlY2ltYWwobnVtKSAtPiBudW1cbiAgY29uc3QgYmFzZSA9IERlY2ltYWwucm91bmQoXG4gICAgRGVjaW1hbChudW0pICogTWF0aC5wb3coMTBtLCAtMW0gKiBEZWNpbWFsKGV4cG9uZW50KSksXG4gICAge1xuICAgICAgbWF4aW11bUZyYWN0aW9uRGlnaXRzLFxuICAgICAgcm91bmRpbmdNb2RlOiAnaGFsZkV4cGFuZCcsXG4gICAgfVxuICApO1xuICBjb25zdCBzdXBlcnNjcmlwdEV4cG9uZW50ID0gWy4uLmV4cG9uZW50LnRvU3RyaW5nKCldLm1hcCgoY2gpID0+IFNVUEVSU0NSSVBUU1tjaF0pLmpvaW4oJycpO1xuICByZXR1cm4gYCR7YmFzZX0g1yAxMCR7c3VwZXJzY3JpcHRFeHBvbmVudH1gO1xufVxuXG5jb25zb2xlLmxvZyhzY2lubyg2LjAyMjE0MDllKzIzLCA4KSk7XG4iLCJ2aXNpYmxlQ29tcG9uZW50cyI6WyJFZGl0b3IiLCJPdXRwdXQiLCJDb25zb2xlIl0sImRlY2ltYWxJbXBsIjoiZGVjaW1hbCAxMjgiLCJ2aWV3VHlwZSI6ImNvbHVtbnMifQ==) is the program I'm trying to get working — the code that Babel adds to transform the spread operator is much larger than the actual program itself. Maybe we can configure Babel somehow not to downlevel quite so far...